src/modules/core/components/Fields/InputStatus/InputStatus.tsx

```diff
@@ -14,6 +14,7 @@ export interface Appearance {
   statusShema?: 'info';
   size?: 'small' | 'medium';
   textSpace?: 'wrap';
+  align?: 'right';
```

Comment:
I  actually don't think that more values are needed here, similar to other props: direction, textSpace. It's mote for the optional use of this css.

Reply:
I wrote "align" this way, similar like textSpace and statusSchema were written before, because I add this class using getMainClasses function: 
```
<Element
      className={getMainClasses(appearance, styles, {
        error: !!error,
        hidden: !text || (!!error && !isNil(touched) && !touched),
      })}
    >
```
With getMainClasses function I don't have to write condition to add this class.